.github/workflows/nightly.yml

```
@@ -0,0 +1,234 @@
+on:
+  push:
+    branches: [main]
+  schedule:
+    - cron: '0 2 * * *'  # Run at 2 AM UTC daily
+
+env:
+  CARGO_TERM_COLOR: always
+
+permissions:
+  contents: write
+  releases: write
```

actionlint (1.7.7)

unknown permission scope "releases". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)